### PR TITLE
Finished fluent configuration

### PR DESCRIPTION
**Pull request by rinchan603**

### CODE REVIEW

**Review by 4500zenja1 — APPROVED** [review]
Yeah, I think nothing is goes from THOSE configurations, so I can approve that. :)

File: Elevel/Persistence/Configurations/QuestionConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public class QuestionConfiguration
+    {
+        public void Configure(EntityTypeBuilder<Question> builder)
+        {
+            builder.ToTable("Question");
+
+            builder.HasOne(x => x.Audition)
+                .WithMany(x => x.Questions)
+                .HasForeignKey(x => x.AuditionId);
```
Comment (parys) [inline Elevel/Persistence/Configurations/QuestionConfiguration.cs:15]: if Audition is not required, so please make AuditionId nullable and mark here as .IsRequired(false)

Comment (rinchan603) [reply Elevel/Persistence/Configurations/QuestionConfiguration.cs]: Done

File: Elevel/Elevel.Infrastructure.Services/Implementation/UserService.cs [hunk 2]
```diff
@@ -43,6 +43,7 @@ public async Task<AuthenticationModel> GetTokenAsync(TokenRequestModel model)
                 authenticationModel.Email = user.Email;
                 authenticationModel.FirstName = user.FirstName;
                 authenticationModel.LastName = user.LastName;
+                authenticationModel.Avatar = "..\\Elevel.Api\\Resources\\default_avatar.jpg";
```
Comment (parys) [inline Elevel/Elevel.Infrastructure.Services/Implementation/UserService.cs:46]: bad practice to show path + in deployed version you will not have this folder. Please create wwwroot folder in root of WebApi project

**Review by parys — APPROVED** [review]

**Review by ZokirS — APPROVED** [review]

File: Elevel/Elevel.Infrastructure.Services/Implementation/UserService.cs (hunk 2 above)
Comment (rinchan603) [reply Elevel/Elevel.Infrastructure.Services/Implementation/UserService.cs:46]: Done

**Pull request merged**